docs/api/web-frame.md

```diff
@@ -57,26 +57,38 @@ Sets the maximum and minimum pinch-to-zoom level.
 
 Sets the maximum and minimum layout-based (i.e. non-visual) zoom level.
 
-### `webFrame.setSpellCheckProvider(language, autoCorrectWord, provider)`
+### `webFrame.setSpellCheckProvider(language, provider)`
 
 * `language` String
-* `autoCorrectWord` Boolean
 * `provider` Object
-  * `spellCheck` Function - Returns `Boolean`.
-    * `text` String
+  * `spellCheck` Function.
+    * `words` String[]
+    * `callback` Function
+      * `misspeltWords` String[]
```

Comment:
curious if callback accepts `Array<boolean>` instead of returning word itself. i.e,

```
spellcheck: (words, callback) => {
 // do some async
 callback(words.map(isMisspelled));
}
```

Reply:
Currently it doesn't, but I can make changes to change the API to accept `Array<boolean>` if we want.
What would be the benefit of doing that?

Reply:
That sounds reasonable. However, to do that, we will have to maintain the list of all words on the c++ side and then map the returned array to those to get the locations in the text to return back to blink. I was just trying to avoid running through all the words by using a map. So, it's definitely doable but I'm not sure how important it is.
Also, APIwise returning misspelt words seems better than an array of booleans. But that's just me. 😃 
I'd like to see if others have any opinions about this. I'm okay going either way!

Reply:
/cc @juturu